SUMMARY.md

```diff
@@ -99,6 +99,13 @@
         * [Import HTML composition](guides/web-apps/import-html-composition/README.md)
         * [Avoiding URI conflicts](guides/blending/avoiding-uri-conflicts/README.md)
         * [Avoiding CSS conflicts](guides/blending/avoiding-css-conflicts/README.md)
+    * [Publishing in the App Warehouse](guides/publishing/README.md)
```

Comment:
I will go with "Publishing to the App Warehouse", because "App Warehouse" is the name we use everywhere else.